packages/buck-util/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@textile/buck-util",
-  "version": "1.0.4",
-  "description": "NodeJS utiliting for pushing and pulling buckets.",
+  "version": "1.0.5-alpha.0",
+  "description": "This library has been replaced by @textile/buckr. This library is deprecated.",
```

Comment:
name always bugged me so took the chance to get rid of this old one

Reply:
LOL, cool!

Reply:
buckr up!